CHANGELOG.md

```diff
@@ -26,9 +26,10 @@
 - PR #992 Add comparison binaryops to DateTime columns
 - PR #996 Replace relative path imports with absolute paths in tests
 - PR #995 CSV Reader: Add index_col parameter to specify the column name or index to be used as row labels
-- PR #1007 Replace relative path imports with absolute paths in cudf
 - PR #997 Add property index setter
+- PR #1007 Replace relative path imports with absolute paths in cudf
```

Comment:
This shouldn't be here...

Reply:
The PR's got out of order on the changelog, not sure. Can revert it if needed.

Reply:
Ah sorry missed the deletion, this is fine.